<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EphemerisData</class>
 <widget class="QWidget" name="EphemerisData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1536</width>
    <height>865</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1535</width>
    <height>865</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>星历数据参数</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="6" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>偏心率：</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>平近点角(度)：</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>升交点赤经(度)：</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>倾角(度)：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>卫星名称：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>瞬跟历元日期：</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>近地点幅角(度)：</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>半长轴：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>任务代号：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="satelliteIdentification"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="satelliteName"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>卫星标识：</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>瞬跟历元时刻：</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0" colspan="2">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="satellitCode"/>
        </item>
        <item row="3" column="1">
         <widget class="QDateEdit" name="dateOfEpoch"/>
        </item>
        <item row="4" column="1">
         <widget class="QTimeEdit" name="timeOfEpoch"/>
        </item>
        <item row="5" column="1">
         <widget class="QDoubleSpinBox" name="semimajorAxis"/>
        </item>
        <item row="6" column="1">
         <widget class="QDoubleSpinBox" name="eccentricity"/>
        </item>
        <item row="7" column="1">
         <widget class="QDoubleSpinBox" name="dipAngle"/>
        </item>
        <item row="8" column="1">
         <widget class="QDoubleSpinBox" name="rightAscension"/>
        </item>
        <item row="9" column="1">
         <widget class="QDoubleSpinBox" name="argumentOfPerigee"/>
        </item>
        <item row="10" column="1">
         <widget class="QDoubleSpinBox" name="meanAnomaly"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addBtn">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editBtn">
       <property name="text">
        <string>编辑</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delBtn">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refreshBtn">
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
